master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -882,11 +882,12 @@ private[celeborn] class Master(
     rpcEnv.awaitTermination()
   }
 
-  override def close(): Unit = synchronized {
+  override def close(graceful: Boolean): Unit = synchronized {
```

Comment:
seems `stop` is a better name.

Reply:
> seems `stop` is a better name.

Yea, but Master extends RpcEndpoint it already have a stop method.